etcd/etcd_test.go

```
@@ -310,6 +311,50 @@ func TestReceiveInsert(t *testing.T) {
 	}
 }
 
+func TestReceiveExpire(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	if err := etcdtest.DeleteAll(); err != nil {
+		t.Error(err)
+		return
+	}
+
+	c, err := New(Options{etcdtest.Urls, "/skippertest", 0, false, "", "", ""})
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	c.LoadAll()
```

Error is not checked here

---

Error is checked now.